src/composite/MegaChart.js

```diff
@@ -60,6 +60,7 @@
     MegaChart.prototype.publish("titleBackgroundColor", null, "html-color", "Background Color", null, { tags: ["Intermediate"], optional: true });
     MegaChart.prototype.publish("maximizedBackgroundColor", '#FFFFFF', "html-color", "Background Color while maximized", null, { tags: ["Intermediate"], optional: true });
 
+    MegaChart.prototype.publish("useParamFlyoutTitles", false, "boolean", "useParamFlyoutTitles");
```

Comment:
I am not sure this describes what is happening when set to true?  Its more a "onlyShowParamsInTitle"?